src/components/Plugin/DaoModule/CustomBlock.vue

```diff
@@ -13,26 +13,50 @@
         <PluginDaoModuleTransactionPreview :transaction="tx" />
       </div>
     </div>
-    <UiButton
+    <div
       v-if="questionDetails?.questionId"
-      @click="showQuestion"
-      class="width-full mb-2"
+      class="mb-3 p-4 border rounded-2 text-white text-center"
     >
-      Show Question
-    </UiButton>
+      <div>
+        <Label> {{ approvalData.decision }} </Label>
+      </div>
+      <div v-if="questionState === 3">
```

Comment:
I feel that this can be changed to `questionState === QuestionStates.waitingForApproval` or something like that? I wasn't able to access to the `QuestionStates` but I am open to any feedback

Reply:
If you want to get access to `QuestionStates` const in the vieww, you need to add it on the data() function here  https://github.com/snapshot-labs/snapshot/pull/376/files/700f34119670eb0428c99a4830b49ab614fa3e55#diff-6183acbc23bece148aa1ff1517af765744b4894675b1077c513185198106fc9eR82